test/collect_coverage_test.dart

```diff
@@ -179,6 +179,11 @@ void main() {
       28: 'fooAsync',
       38: 'isolateTask'
     });
+    if (platformVersionCheck(2, 17)) {
```

Comment:
As above, can this be a separate test, or can you pass `skip: !platformVersionCheck(2, 17)` to the `expect` call?

Reply:
Done